### PR TITLE
Feature homepage

### PR DESCRIPTION
**Pull request by mhhutton**
-Updates components
-Updates login function in Nav bar,
-Updates with simple non-functioning search bar in navigation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+<p></p>Jest: 27.0.4 <img src="https://www.vectorlogo.zone/logos/jestjsio/jestjsio-icon.svg" alt="jest" width="40" height="40"/>
+<p></p>Cypress: 7.4 <img src="https://raw.githubusercontent.com/simple-icons/simple-icons/6e46ec1fc23b60c8fd0d2f2ff46db82e16dbd75f/icons/cypress.svg" alt="cypress" width="40" height="40"/>
+<p>Express: 4.17.1 <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/express/express-original-wordmark.svg" alt="express" width="40" height="40"/></p>
+<p>mongoose: 5.13.2 <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/mongodb/mongodb-original-wordmark.svg" alt="mongodb" width="40" height="40"/></p>
+<p>bable <img src="https://www.vectorlogo.zone/logos/babeljs/babeljs-icon.svg" alt="babel" width="40" height="40"/></p>
+<p>webpack <img src="https://raw.githubusercontent.com/devicons/devicon/d00d0969292a6569d45b06d3f350f463a0107b0d/icons/webpack/webpack-original-wordmark.svg" alt="webpack" width="40" height="40"/></p>
+
+## Setup ##
+<p>To run the project</p>
+<p>-npm install</p>
```
Comment (Pix3lAssassin) [inline README.md]: appears to be an extra dash and missing .env-example explanation

File: client/dist/index.html [hunk 2]
```diff
@@ -3,7 +3,9 @@
 <head>
   <title>NextStop</title>
   <link rel="stylesheet" href="styles.css">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Oswald&display=swap" rel="stylesheet">
```
Comment (Pix3lAssassin) [inline client/dist/index.html]: Should probably indent this

File: client/src/components/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  Route,
+  Link,
+} from 'react-router-dom';
+import {
+  Tab, Paper, Tabs,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import Events from '../Events/Events.jsx';
+import Flights from '../Flights/Flights.jsx';
+import Hotels from '../Hotels/Hotels.jsx';
```
Comment (Pix3lAssassin) [inline client/src/components/Home/Home.jsx:14]: These appear to be unused in this file

File: client/src/components/Navigation.jsx [hunk 4]
```diff
@@ -1,22 +1,114 @@
...
+    justifyContent: 'space-evenly',
+    alignItems: 'center',
+    color: 'white',
+  },
+  loginBtn: {
+    color: 'white',
+    border: '2px solid white',
+    borderRadius: '8px',
+    marginLeft: '5px',
+  },
```
Comment (Pix3lAssassin) [inline client/src/components/Navigation.jsx:66]: Could probably extract this out to a new file

File: client/src/index.jsx [hunk 5]
```diff
@@ -1,23 +1,36 @@
 import React, { useState, useEffect } from 'react';
 import ReactDOM from 'react-dom';
 
-import { Container, CssBaseline } from '@material-ui/core';
-import { createTheme, ThemeProvider } from '@material-ui/core/styles';
+import { CssBaseline } from '@material-ui/core';
+// import { createTheme, createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
```
Comment (Pix3lAssassin) [inline client/src/index.jsx]: unnecessary comment

File: client/src/index.jsx [hunk 6]
```diff
@@ -1,23 +1,36 @@
...
 import AppContext from './helpers/context';
 
-const theme = createTheme({
-  palette: {
-    type: 'dark',
+// const theme = createMuiTheme({
+//   palette: {
+//     type: 'dark',
+//   },
+// });
```
Comment (Pix3lAssassin) [inline client/src/index.jsx]: unnecessary comment

**Review by Pix3lAssassin — APPROVED** [review]

**Review by calebbratton — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mhhutton) [reply README.md]: removed dash, will add .enviexample later

File: client/src/components/Home/Home.jsx (hunk 3 above)
Comment (mhhutton) [reply client/src/components/Home/Home.jsx:14]: Still weeding out what we are using in the future.

File: client/src/index.jsx (hunk 6 above)
Comment (mhhutton) [reply client/src/index.jsx]: deleted comments

File: client/src/index.jsx (hunk 5 above)
Comment (mhhutton) [reply client/src/index.jsx]: deleted comments

File: client/dist/index.html (hunk 2 above)
Comment (mhhutton) [reply client/dist/index.html]: properly indented

**Pull request merged**